packages/@aws-cdk-testing/framework-integ/test/aws-backup/test/integ.backup.ts

```diff
@@ -46,9 +57,6 @@ class TestStack extends Stack {
         moveToColdStorageAfter: Duration.days(30),
         deleteAfter: Duration.days(120),
       }],
-      recoveryPointTags: {
-        stage: 'prod',
-      },
```

Comment:
Why is this removed from the existing integ test?

Reply:
I don't remember the intention at the time, but it might be by my mistake. These lines were introduced few days before I submitted this PR, so I guess that some miss operation in my local environment at the time. Anyway, thank you for your nice catching! I'll revert this change and regenerate snapshots.